packages/apify/src/utils.ts

```diff
@@ -6,6 +6,8 @@ import semver from 'semver';
 // @ts-expect-error if we enable resolveJsonModule, we end up with `src` folder in `dist`
 import { version as apifyClientVersion } from 'apify-client/package.json';
 // @ts-expect-error if we enable resolveJsonModule, we end up with `src` folder in `dist`
+import { version as crawleeVersion } from 'crawlee/package.json';
```

Comment:
we should check it from the core package i guess, the meta package might not be there and this would fail

Reply:
Hrm, good point. We'll need a release of /core that exports the package.json first (I'll open that rn) then we can do this!